### ISSUE TITLE
Support transfer with MetaMask

### ISSUE THREAD

**Issue by zjb0807** [AcalaNetwork/bodhi.js#152]
https://github.com/AcalaNetwork/Acala/pull/1644

https://github.com/AcalaNetwork/Acala/blob/82d54ff5986149c07a604ae8d119685d77e6533f/ts-tests/tests/test-sign-eth.ts#L58-L66

**Comment by zjb0807** [comment]
```
{
    id: 8572753427450
    jsonrpc: "2.0"
    error: {
      "code": -32601,
      "message": "EIP-1559 transactions not implemented (method=\"EIP-1559 transactions\", provider=\"eth-providers\", code=NOT_IMPLEMENTED, version=bodhi.js/providers/2.1.6)"
    }
}
```

**Comment by zjb0807** [comment]
EIP-1559 duplicate with #174 
Closed by #167
